server/src/test/java/org/opensearch/action/admin/cluster/node/stats/NodeStatsTests.java

```diff
@@ -480,6 +481,17 @@ public void testSerialization() throws IOException {
                     assertEquals(replicationStats.getTotalBytesBehind(), deserializedReplicationStats.getTotalBytesBehind());
                     assertEquals(replicationStats.getMaxReplicationLag(), deserializedReplicationStats.getMaxReplicationLag());
                 }
+
+                if (nodeIndicesStats == null) {
+                    assertNull(deserializedNodeIndicesStats);
+                } else {
+                    UnreferencedFileCleanUpStats cleanUpStats = nodeIndicesStats.getMerge().getTotalCleanUpStats();
+                    UnreferencedFileCleanUpStats deserializedCleanUpStats = deserializedNodeIndicesStats.getMerge().getTotalCleanUpStats();
+                    assertEquals(
+                        cleanUpStats.getTotalUnreferencedFileCleanupCount(),
+                        deserializedCleanUpStats.getTotalUnreferencedFileCleanupCount()
+                    );
```

Comment:
We are duplicating the if block. Let us move the else block logic to the above else block.

Reply:
Addressed the comment.